x-pack/platform/plugins/private/upgrade_assistant/server/lib/reindexing/reindex_service.ts

```diff
@@ -169,7 +169,12 @@ export const reindexServiceFactory = (
    * @param reindexOp
    */
   const setReadonly = async (reindexOp: ReindexSavedObject) => {
-    const { indexName } = reindexOp.attributes;
+    const { indexName, rollupJob } = reindexOp.attributes;
+
+    if (rollupJob) {
+      await esClient.rollup.stopJob({ id: rollupJob, wait_for_completion: true });
```

Comment:
What happens when it fails to stop the job? Do we provide users with actionable information about the issue like a manual workaround?

Reply:
I think this gets into the expectations surrounding this code. If any other aspect of reindexing fails, do we tell the users about a manual work around?

There's not much reason to expect this to fail. Actually, I'm not sure I'd have much advice for the user if this failed since I'm unfamiliar with what could cause it to fail.

Reply:
Yeah\ generally we return the error message. My question was more about what the content of a failure message is and is it something users can self-service? Generally UA reindexing error messages are self-servicable but I'm not familiar with common failure conditions here so I thought I'd ask.

It might be the case that these are, for example, retryable or something we can handle.

Reply:
I checked with @martijnvg and he stated that there's no reason to anticipate any error states when stopping rollups so I think I'll leave the code as is.